### ISSUE TITLE
wrong forced autocomplete

### ISSUE THREAD

**Issue by rajinder-yadav** [Microsoft/vscode#40793]
- VSCode Version: Code - Insiders 1.20.0-insider (08ec16001adaab8a86183431f6aec09209caa4ab, 2017-12-15T06:08:07.643Z)
- OS Version: Darwin x64 17.3.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
markdown-toc|Ala|1.5.6
vscode-markdownlint|Dav|0.12.0
EditorConfig|Edi|0.11.1
vscode-eslint|dba|1.4.3
tslint|eg2|1.0.24
prettier-vscode|esb|1.0.1
cpptools|ms-|0.14.5
debugger-for-chrome|msj|4.0.0
vim|vsc|0.10.6
markdown-pdf|yza|0.1.7

(2 theme extensions excluded)

---

Steps to Reproduce:

1. type "toB"
1. See pic below of suggestion displayed.
1. type "toBe("

![screen shot 2017-12-24 at 9 35 36 pm 3](https://user-images.githubusercontent.com/439226/34331777-ecf589ec-e8f7-11e7-9138-82d502358029.png)

Incorrectly autocompletes "toBeTruthy()". Also notice incorrect highlight is suggested after step 1, when it should suggest (select) text at top.

Reproduces without extensions: Yes/No

**Comment by mjbvz** [comment]
@rajinder-yadav Is `toBeTruthy` the suggestion selected after you type `.toBe`? 

You should also be able to disable the auto-accept by setting: ` "editor.acceptSuggestionOnCommitCharacter": false`

**Comment by rajinder-yadav** [comment]
Yes.

**Comment by mjbvz** [comment]
@rajinder-yadav I can't repo with:

```js
import 'jasmine';

expect(1).toBe
```

<img width="688" alt="screen shot 2018-01-05 at 2 49 46 pm" src="https://user-images.githubusercontent.com/12821956/34631928-b4b85440-f227-11e7-8fa6-92710e03b57f.png">

Please provide proper repo steps with a simplified code example

**Comment by rajinder-yadav** [comment]
Creating a app using angular-cli and then in the test file with extension .spec try to repro the issues. This happened for me inside an Angular app.

**Comment by mjbvz** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/41060

You can disable completing on `(` by setting `"editor.acceptSuggestionOnCommitCharacter": false`
